platform/os.go

```diff
@@ -4,8 +4,9 @@ import (
 	"bufio"
 	"fmt"
 	"io"
-	"log"
 	"os"
+
+	"github.com/Azure/azure-container-networking/log"
```

Comment:
can you make this change when you do for platform package?

Reply:
+1